### PR TITLE
Remove push channel from Message Center 

### PR DESCRIPTION
**Pull request by amorimluzia**
Remove push channel from Message Center by creating a new GN

### CODE REVIEW

File: collections/retail/general-notifications/event-general-notifications/message-received/event-general-notifications.json [hunk 1]
```diff
@@ -45,6 +46,31 @@
...
+          "channelSettings": "/new-message-received-mailout/in-app-notification/in-app_channel-settings.json",
+          "isEnabledByDefault": true,
+          "isManageableByUser": false,
+          "routeInfo": {
+            "whereTo": "conversation-view",
+            "data": "{\"id\":\"{{event.conversationId}}\"}"
+          },
+          "engagementTemplates": [
+            {
+              "path": "/new-message-received-mailout/in-app-notification/en_US/engagement-template.json",
```
Comment (afsinka-backbase) [inline collections/retail/general-notifications/event-general-notifications/message-received/event-general-notifications.json:67]: do you need to create `in-app-notification` folder under `new-message-received-mailout`?

Comment (amorimluzia) [reply collections/retail/general-notifications/event-general-notifications/message-received/event-general-notifications.json:67]: thanks @afsinka-backbase I missed it during the copy/ paste. It is fixed now

File: collections/business/general-notifications/event-general-notifications/message-received/event-general-notifications.json [hunk 2]
```diff
@@ -45,6 +46,31 @@
...
+          "channelSettings": "/new-message-received-mailout/in-app-notification/in-app_channel-settings.json",
+          "isEnabledByDefault": true,
+          "isManageableByUser": false,
+          "routeInfo": {
+            "whereTo": "conversation-view",
+            "data": "{\"id\":\"{{event.conversationId}}\"}"
+          },
+          "engagementTemplates": [
+            {
+              "path": "/new-message-received-mailout/in-app-notification/en_US/engagement-template.json",
```
Comment (afsinka-backbase) [inline collections/business/general-notifications/event-general-notifications/message-received/event-general-notifications.json:67]: @amorimluzia same here I guess

Comment (amorimluzia) [reply collections/business/general-notifications/event-general-notifications/message-received/event-general-notifications.json:67]: yes, I added a few minutes ago :)

File: collections/business/general-notifications/event-general-notifications/message-received/event-general-notifications.json [hunk 3]
```diff
@@ -45,6 +46,31 @@
...
+    },
+    {
+      "id": "new-message-received-mailout",
+      "name": "New Message Received Mailout",
+      "userPreferenceEnabled": false,
+      "additionalFilterLogic": "#triggeredEvent['origin'] != null && #triggeredEvent['origin'] == 'engagement'",
+      "deliveryOptions": [
+        {
+          "channel": "in-app-notification",
+          "channelSettings": "/new-message-received-mailout/in-app-notification/in-app_channel-settings.json",
```
Comment (afsinka-backbase) [inline collections/business/general-notifications/event-general-notifications/message-received/event-general-notifications.json:58]: @amorimluzia sorry, this one also should be under `in-app-notification` folder I guess, right? 😅

Comment (amorimluzia) [reply collections/business/general-notifications/event-general-notifications/message-received/event-general-notifications.json:58]: yes, I just added it, seconds ago

**Review by afsinka-backbase — APPROVED** [review]

**Review by madz0 — APPROVED** [review]

**Pull request merged**